tests/helpers.py

```diff
@@ -636,7 +636,7 @@ def validate_cephfilesystem(fs_name):
     return True if (ceph_validate and ocp_validate) else False
 
 
-def get_all_storageclass_names():
+def get_all_storageclass_names(all_ns=False):
```

Comment:
as mentioned above, please move this function to somewhere under ocs_ci dir. 
Please make sure I'm right but I don't see any usages of this function so it should be smooth.